.travis.yml

```diff
@@ -11,9 +11,7 @@ addons:
       - apache2
 
 php:
-  - 5.6
-  - 7.0
-  - 7.1
+  - 7.2
```

Comment:
On line 23, in the befor_script section. NPM is installed at a pinned version. Maybe set this to the latest version 10 in order to get latest and greatest? Might save some additional update time in the future.